openapi2beans/JavaChecker/src/main/java/dev/galasa/openapi2beans/example/generated/BeanWithArray.java

```diff
@@ -0,0 +1,22 @@
+package dev.galasa.openapi2beans.example.generated;
```

Comment:
skipping my review of the rest of the generated files because I'm not convinced this code should be checked-in yet.